test/test_api/test_api.py

```diff
@@ -449,3 +451,92 @@ def test_do_dummy_prediction(dask_client, fit_dictionary_tabular):
     estimator._clean_logger()
 
     del estimator
+
+
+@pytest.mark.parametrize("disable_file_output", [True, False])
+@pytest.mark.parametrize('openml_id', (40981,))
+@pytest.mark.parametrize('resampling_strategy', (HoldoutValTypes.holdout_validation,
+                                                 CrossValTypes.k_fold_cross_validation,
+                                                 ))
+def test_pipeline_fit(openml_id, resampling_strategy, backend, disable_file_output):
+    # Get the data and check that contents of data-manager make sense
+    X, y = sklearn.datasets.fetch_openml(
+        data_id=int(openml_id),
+        return_X_y=True, as_frame=True
+    )
+    X_train, X_test, y_train, y_test = sklearn.model_selection.train_test_split(
```

Comment:
Let us make this function faster with a 20% for train and 80% for testing

Reply:
okay